src/pca.jl

```diff
@@ -98,7 +98,7 @@ end
 
 function pcacov{T<:FloatingPoint}(C::DenseMatrix{T}, mean::Vector{T};
                 maxoutdim::Int=size(C,1),
-                pratio::T=default_pca_pratio)
+                pratio=convert(T, default_pca_pratio))
```

Comment:
Not only the routines should work with the default ```pratio```; it should also work with the explicitly specified ```pratio```. But I suppose it's better here (and in the functions below) to just pass ```pratio``` as-is to the functions where it is actually used (i.e. ```check_pcaparams()``` and ```choose_pcadim()```) and do the conversion there.